patches/dompurify+2.3.4.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/node_modules/dompurify/dist/purify.js b/node_modules/dompurify/dist/purify.js
+index 68bc40f..03bd803 100644
+--- a/node_modules/dompurify/dist/purify.js
++++ b/node_modules/dompurify/dist/purify.js
+@@ -623,6 +623,12 @@
+ 
+     var HTML_INTEGRATION_POINTS = addToSet({}, ['foreignobject', 'desc', 'title', 'annotation-xml']);
+ 
++    // Certain elements are allowed in both SVG and HTML
++    // namespace. We need to specify them explicitly
++    // so that they don't get erronously deleted from
++    // HTML namespace.
++    var COMMON_SVG_AND_HTML_ELEMENTS = addToSet({}, ['title', 'style', 'font', 'a', 'script']);
```

Comment:
This was moved to a constant, as it was getting re-declared for every element being sanitized which caused a lot of additional object creation which in turn triggered the GC more often.